Integrations/integration-DuoAdminApi.yml

```diff
@@ -70,7 +70,7 @@ script:
 
     OPTIONS_TO_TIME = {
         '10_seconds_ago' : datetime.datetime.now() - datetime.timedelta(seconds = 10),
-        '1_minutes_ago' : datetime.datetime.now() - datetime.timedelta(minutes = 1),
```

Comment:
Since we've already released leave this option too for backwards compatibility. So if there is a playbook using this already we will still parse this value fine. in the predefined it is ok to remove as you did below.

Reply:
Done